src/Standards/Generic/Tests/CodeAnalysis/JumbledIncrementerUnitTest.1.inc

```diff
@@ -0,0 +1,83 @@
+<?php
```

Comment:
For the "Skip for loop without body" part (~line 66), you've now added a test with a parse error, which is good, but there is actually valid code which could also trigger that code. I think it would be good to add a test with valid code too.

```php
for ($i = 0; $i < 20; $i++);
```

Possibly also add a test where the outer loop has the scope opener/closer, but the inner loop does not ?

Reply:
I added a new commit with the two tests.